lib/srv/ingress/reporter.go

```diff
@@ -76,16 +79,34 @@ var (
 	authenticatedConnectionsAccepted = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "teleport",
 		Name:      "authenticated_accepted_connections_total",
-	}, commonLabels)
+	}, []string{"ingress_path", "ingress_service", "service_metadata"})
 
 	// authenticatedConnectionsActive measures the current number of active connections that
 	// successfully authenticated.
 	authenticatedConnectionsActive = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: "teleport",
 		Name:      "authenticated_active_connections",
-	}, commonLabels)
+	}, []string{"ingress_path", "ingress_service", "service_metadata"})
 )
 
+func metadataFromClientCert(clientCert *x509.Certificate) string {
+	identity, err := tlsca.FromSubject(clientCert.Subject, clientCert.NotAfter)
+	if err != nil {
+		return "unknown"
+	}
+
+	return fmt.Sprintf("%v;%v;%v;%v;%v", identity.TeleportCluster, identity.RouteToCluster, identity.KubernetesCluster, identity.RouteToApp.Name, identity.RouteToDatabase.ServiceName)
```

Comment:
nit:
```suggestion
	return strings.Join([]string{identity.TeleportCluster, identity.RouteToCluster, identity.KubernetesCluster, identity.RouteToApp.Name, identity.RouteToDatabase.ServiceName}, ";")
```

Reply:
Fixed: moved to separate fields; I've also dropped kube/db/app names from the list.